doc/api/util.markdown

```diff
@@ -548,6 +548,16 @@ Deprecated predecessor of `console.log`.
 
 Deprecated predecessor of `console.log`.
 
+## util._extend(obj)
+
+    Stability: 0 - Deprecated: Use Object.assign() instead.
```

Comment:
I suggested a change in https://github.com/nodejs/node/pull/4907. If it gets through, we can make the `Object.assign()` a link.